src/components/rangeInput.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC } from "react"
+
+import Input from "./input/index"
+
+import Label from "./fieldHelpers/label"
+
+interface Props {
+  name: {
+    min: string
+    max: string
+  }
+  handleChangeMin: (value) => void
+  handleChangeMax: (value) => void
+  value?: {
+    min: number
+    max: number
+  }
+  isValid?: {
+    min: boolean
+    max: boolean
+  }
+  placeholder?: {
+    min: string
+    max: string
+  }
+  required?: boolean
+  rangeInputLabel?: string
```

Comment:
Any reason to not just call it label?

Reply:
Yes because Storybook complains : "label" is already one of its args

Reply:
Can we try to solve it in storybook? Right now we make our external interface more complex because the internal documentation tool is not happy with the naming....

Reply:
Yes, I fixed it and called it `label` 👍